packages/components/src/InputDate/InputDate.css

```diff
@@ -0,0 +1,10 @@
+.inputDate {
```

Comment:
Don't need this css

Reply:
https://github.com/GetJobber/atlantis/pull/783/commits/1fcf3e6bf09e639e387122246f800d101d6e02ca